roles/nova-common/templates/etc/nova/nova.conf

```diff
@@ -55,12 +57,12 @@ keys_path=/var/lib/nova/keys
 use_deprecated_auth=false
 auth_strategy=keystone
 use_forwarded_for=true
-keystone_ec2_url=https://{{ endpoints.keystone }}:5001/v2.0/ec2tokens
+keystone_ec2_url={{ endpoints.keystone.url.internal }}/{{ endpoints.keystone.version }}/ec2tokens
```

Comment:
This should be `keystone.url.public` I would think, if it's being advertised to end users.